src/test/scala/uk/gov/hmrc/play/http/HttpGetSpec.scala

```diff
@@ -79,8 +90,43 @@ class HttpGetSpec extends WordSpecLike with Matchers with ScalaFutures with Comm
       verify(testGet.testHook1)(url, "GET", None, dummyResponseFuture)
       verify(testGet.testHook2)(url, "GET", None, dummyResponseFuture)
     }
-
   }
 
+  "HttpGet" should {
+    "return an empty string if the query parameters is empty" in {
+      val expected = "http://test.net"
+      val testGet = new UrlTestingHttpGet(expected)
+      testGet.GET("http://test.net", Seq())
+    }
 
+    "return a url with a single param pair" in {
```

Comment:
These tests are confusing - you should always have an assert in a test! I see the assert is hidden away. Can you please create a new HttpGet stub that stores the urls passed to it that you assert on in the test